admin_app/src/components/TypingAnimation.tsx

```diff
@@ -0,0 +1,30 @@
+import { keyframes } from "@emotion/react";
```

Comment:
This is also under `@/components/Common`

And is used under `DetailsDrawer.tsx` under performance

Reply:
ohh I didnt know that - it was part of playground so I pulled it out. Will remove